### ISSUE TITLE
How to skip the lm model, i want to see the directly model prediction after ctc decoding.

### ISSUE THREAD

**Issue by vigneshgig** [mailong25/self-supervised-speech-recognition#42]
Hi, @mailong25  How to skip the lm model. I want to see direct model prediction after CTC decoding and also I'm getting valid_wer as 100 for every epoch, but the training loss is decreasing. why so?
Note: apart from your config I just added the memory_efficient_fp16 = True, to reduce GPU memory consumption, is this affecting the wer.
Please help me out.
Thanks
[hydra_train.log](https://github.com/mailong25/self-supervised-speech-recognition/files/6139140/hydra_train.log)

**Comment by vigneshgig** [comment]
> Hi, @mailong25 How to skip the lm model. I want to see direct model prediction after CTC decoding and also I'm getting valid_wer as 100 for every epoch, but the training loss is decreasing. why so?
> Note: apart from your config I just added the memory_efficient_fp16 = True, to reduce GPU memory consumption, is this affecting the wer.
> Please help me out.
> Thanks
> [hydra_train.log](https://github.com/mailong25/self-supervised-speech-recognition/files/6139140/hydra_train.log)

Hi , I installed latest fairseq and pytorch 1.7.0 with cuda 10.2 in python 3.7.9 version.
Now the valid_wer is decreasing and also it starts from 83~ .
and also trained loss start from 54.0 comparing older version which starts from 117~
Uploading hydra_train.log…
Please tell, how can i check the predicted text before the lm decoding.

**Comment by mailong25** [comment]
Sorry for the late response, 
In the transcriber, you can set lm_type = 'viterbi' and see what happened, I haven't check if it working properly or not
